pgjdbc/src/main/java/org/postgresql/jdbc/PgClobText.java

```diff
@@ -0,0 +1,218 @@
+/*
+ * Copyright (c) 2004, PostgreSQL Global Development Group
+ * See the LICENSE file in the project root for more information.
+ */
+
+/*
+ * Implementation of Clob interface backed by Postgres varchar/text fields.
+ *
+ * See PgClob.java for implementation using Postgres Large Objects.
+ */
+
+package org.postgresql.jdbc;
+
+import org.postgresql.util.GT;
+import org.postgresql.util.PSQLException;
+import org.postgresql.util.PSQLState;
+
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.io.Reader;
+import java.io.StringReader;
+import java.io.Writer;
+import java.sql.Clob;
+import java.sql.SQLException;
+
+public class PgClobText implements java.sql.Clob {
+
+  private @Nullable String data;
+
+  public PgClobText() {
+    this.data = "";
+  }
+
+  public PgClobText(@Nullable String s) {
+    this.data = s;
+  }
+
+  // This method frees the Clob object and releases the resources the
```

Comment:
For implemented methods the javadoc is inherited. There is really no need to repeat the docs here